docs/products/kafka/howto/prevent-full-disks.rst

```diff
@@ -0,0 +1,81 @@
+Prevent full disks
+===================
+
+To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low.
```

Comment:
What does it mean "running low"? Can we be more precise and state when (like less than 20% of disk space)?

Reply:
Copying here for reference          
        "critical": "used_percent_avg > 97.0", 
        "warning": "used_percent_avg > 90.0"

Reply:
Updated text